README.md

```diff
@@ -234,6 +235,10 @@ def create
   @user = User.create( user_params )
 end
 
+def show
+  @user = User.find( user_params )
+end
+
```

Comment:
Do we need to add this method here? If it doesn't add to the documentation, I'd drop it, leaving it as it was.